packages/dht/src/dht/recursive-operation/RecursiveOperationSession.ts

```diff
@@ -45,6 +46,7 @@ export class RecursiveOperationSession extends EventEmitter<RecursiveOperationSe
     private timeoutTask?: NodeJS.Timeout 
     private completionEventEmitted = false
     private noCloserNodesReceivedCounter = 0
+    private noCloserNodesReceivedFrom: Set<NodeID> = new Set()
```

Comment:
This set is never read from (only written to)?

Reply:
What I mean with this comment is that the set is only `add`:ed to but never read or checked anywhere in the code?